components/NavigationDrawer/src/private/MDCBottomDrawerContainerViewController.m

```diff
@@ -475,12 +484,34 @@ - (void)viewDidDisappear:(BOOL)animated {
 
 - (void)preferredContentSizeDidChangeForChildContentContainer:(id<UIContentContainer>)container {
   [super preferredContentSizeDidChangeForChildContentContainer:container];
+  if ([container isKindOfClass:[UIViewController class]]) {
```

Comment:
is this so we don't issue a change when the header view controller height changes?

Reply:
`UIContentContainer` may be a presentationController so we have to make sure it's a `UIViewController` we can remove the condition `containerViewController == self.contentViewController` if we want the same behavior for both the header and content.